chia/wallet/wallet_state_manager.py

```diff
@@ -1149,11 +1158,41 @@ async def handle_clawback(
         if is_recipient is not None:
             spend_bundle = SpendBundle([coin_spend], G2Element())
             memos = compute_memos(spend_bundle)
+            spent_height: uint32 = uint32(0)
+            if coin_state.spent_height is not None:
+                self.log.debug("Resync clawback coin: %s", coin_state.coin.name().hex())
```

Comment:
In the normal case, if the clawback coin is spent, this handle_clawback will not be executed. It will only be spent here when the wallet first time saw the coin, and it is spent. The tx record shouldn't exist at this point since if it exists the wallet should also have a related coin record. If there is a coin record, this handle_clawback will not be executed.